grout_server/settings.py

```diff
@@ -164,3 +171,10 @@
 
 # Grout-specific global variables
 GROUT = { 'SRID': 4326 }
+
+# Try to import extra deployment-specific settings.
+if DEBUG is False:
+    try:
+        from grout_server.settings_deployment import *
```

Comment:
I'll let ops weigh in more fully, but we've used a similar approach in the past on other projects, so I think this is probably on the right track.